src/mac/mod.rs

```diff
@@ -4,9 +4,4 @@ pub mod beacon;
 pub mod command;
 pub mod frame;
 
-pub use frame::{
-    Address, DecodeError, ExtendedAddress, Frame, FrameContent, Header, PanId, ShortAddress,
-    WriteFooter,
-};
-
-pub use frame::frame_control::{AddressMode, FrameControl, FrameType, FrameVersion};
```

Comment:
I'd prefer if all the important types still were re-exported. Without the re-exports, using the library becomes less convenient, and I don't think the internal structure of the module is important to the user.

This is also a breaking change, which I'd like to avoid unless there is a clear benefit.